Templates/EventHubTrigger-CSharp/function.json

```diff
@@ -6,7 +6,8 @@
             "name": "myEventHubMessage",
             "direction": "in",
             "path": "samples-workitems",
-            "connection": ""
+            "connection": "",
+            "consumerGroup": "$Default"
```

Comment:
The runtime will default this if not specified. Should I put these default values here or remove them and let the template take care of it?